source/includes/_advanced-usage.md

```diff
@@ -0,0 +1,38 @@
+#Advanced usage
+
+## Opening product dialog programatically
+
+Tipser product dialog can be openend programatically by using the underlying [Tipser SDK](#tipser-sdk) instance.
```

Comment:
afaik only old dialog you can open in this way :(

Reply:
That's actually very fair point, that getting rid of old product dialog is more complicated than we thought so initially.

Reply:
@mateuszsikora you are right. However the old dialog is so far the only official one and customers are not aware of the new dialog. When the new dialog will become the official one, I will add clarifications to this section.

Reply:
:+1: